pkg/combustion/time.go

```diff
@@ -39,7 +39,17 @@ func configureTime(ctx *image.Context) ([]string, error) {
 func writeTimeCombustionScript(ctx *image.Context) error {
 	timeScriptFilename := filepath.Join(ctx.CombustionDir, timeScriptName)
 
-	data, err := template.Parse(timeScriptName, timeScript, ctx.ImageDefinition.OperatingSystem.Time)
+	values := struct {
```

Comment:
I don't have anything against this but was just wondering why not pass the object as before?

Reply:
Largely just consistency and personal preference. In many other places (at least the ones I wrote), we do a custom struct. I hate the templating syntax and would rather the data shuffling in Go instead of having to access nested variables in the template.